src/main/webapp/js/projects/react/components/ExpandedProject.js

```diff
@@ -8,6 +8,10 @@ import {expandedProjectType} from '../prop_types.js';
  * @return {React.Component} Returns the React component.
  */
 export default function ExpandedProject(props) {
+  React.useEffect(() => {
```

Comment:
Wow, first time seeing a react hook in the wild. Heard a lot about them and how they're a game-changer but haven't used it before.

Reply:
I saw them while browsing around on React documentation, so I thought I'd try them here! It's definitely quite nice to be able to add this kind of behavior without expanding this into a class.